gui/default/index.html

```diff
@@ -344,40 +344,132 @@ <h3 class="panel-title">
+                    <span translate>Log in with WebAuthn</span>
+                  </button>
+                </div>
+
+                <p class="text-info" ng-if="!webauthnAvailable()" translate>
+                  Your browser does not support WebAuthn.
+                </p>
+                <div ng-if="locationDoesNotMatchWebauthnRpId()">
+                  <p class="text-info" translate>
+                    Current domain does not match WebAuthn configuration.
+                  </p>
+                  <p>
+                    <a href="{{docsURL('users/webauthn#webauthn-custom-gui-address')}}" translate>
+                      Syncthing Documentation: Customizing the GUI address
+                    </a>
+                  </p>
+                </div>
+                <p class="text-danger" ng-if="webauthn.errors.notRegistered" translate>
+                  This device or security key is not registered. Please try a different one.
+                </p>
+                <p class="text-info" ng-if="webauthn.errors.aborted" translate>
+                  Authentication aborted by user.
+                </p>
+                <p class="text-danger" ng-if="webauthn.errors.notAllowed" translate>
+                  Authentication failed, please try again.
+                </p>
+                <div ng-if="webauthn.errors.securityError">
+                  <p class="text-danger" translate>
+                    Initialization blocked by browser. Please make sure Syncthing is correctly configured.
+                  </p>
+                  <p>
+                    <a href="{{docsURL('users/webauthn#webauthn-custom-gui-address')}}" translate>
+                      Syncthing Documentation: Customizing the GUI address
+                    </a>
+                  </p>
+                </div>
+                <p class="text-danger" ng-if="webauthn.errors.initFailed" translate>
+                  Initialization failed, see Syncthing logs for details.
+                </p>
+                <p class="text-danger" ng-if="webauthn.errors.authenticationFailed" translate>
+                  Authentication failed, see Syncthing logs for details.
+                </p>
+                <p class="text-info" ng-if="webauthn.errors.noCredentials" translate>
+                  No WebAuthn credentials configured.
+                </p>
+                <p class="text-danger" ng-if="webauthn.errors.uvRequired" translate>
+                  User verification is required, but was not performed.
+                </p>
+
+                <button
+                  ng-if="inferWebauthnAddress() && locationDoesNotMatchWebauthnRpId()"
+                  type="button"
+                  class="btn btn-primary"
+                  ng-click="reloadLoginAtWebauthnAddress()"
+                  translate
+                  translate-value-webauthn-address="{{ inferWebauthnAddress() }}"
+                >
+                  Reload page at {%webauthnAddress%}
+                </button>
```

Comment:
Un-line-break, please, as discussed in the other PR.

Reply:
Done in da4cd0d5c23a19869eaabcc0094d2b1bee6d7019.